Marlin/temperature.cpp

```diff
@@ -394,6 +426,16 @@ uint8_t Temperature::soft_pwm_amount[HOTENDS],
         #endif
 
         temp_ms = ms + 2000UL;
+
+        #if WATCH_THE_BED || WATCH_HOTENDS
+          if (!heated && input > next_watch_temp) {
+            if (input > watch_temp_target) heated = true;
```

Comment:
The original line had me confused. This accomplishes the same thing without being too much like my usual obfuscated coding style.